crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -161,6 +161,7 @@ pub fn blob_preview_and_save_ui(
                             blob,
                             media_type,
                             ctx.app_options().video_decoder_settings(),
+                            ctx.render_ctx().active_frame_idx(),
```

Comment:
We should check and document wether or not this increments each _frame_ or each _pass_

Reply:
this increments each time we call `begin_frame` on `re_renderer`. Which I think we do on all egui passes...? A bit concerning as that's quite wasteful :/

Reply:
We only run a second pass if `request_discard` is called, which is very rare. It currently only happens when a new ListItem scope appears for the first time, and on some interactions with `egui_table` (e.g. when auto-sizing it).